### PR TITLE
Setting up new shop theme

### PR DESCRIPTION
**Pull request by AndyEPhipps**

### CODE REVIEW

**Comment by AndyEPhipps** [comment]
Don't merge yet, just realised I needed to add a few things

File: Gruntfile.js [hunk 1]
```diff
@@ -321,6 +364,9 @@ module.exports = function (grunt) {
       },
       sr18: {
         src: ['dist/css/kss/sr.css', 'dist/css/themes/sr/**/*.css']
+      },
+      shop18: {
+        src: ['dist/css/kss/shop18.css', 'dist/css/themes/sr/**/*.css']
```
Comment (Heleen-cr) [inline Gruntfile.js]: `dist/css/themes/sr/**/*.css`  should be shop instead of sr

File: sass/kss/shop.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+		height: 50px;
+		display: block;
+		background: url(/kss-assets/images/pattern_logo_sr.png) left top no-repeat;
+		background: url(/kss-assets/images/pattern_logo_sr.png) left top no-repeat;
+		-webkit-background-size: contain;
+  	-moz-background-size: contain;
+  	-o-background-size: contain;
+  	background-size: contain;
+	}
+}
```
Comment (Heleen-cr) [inline sass/kss/shop.scss]: new line... sorry :)

File: Gruntfile.js (hunk 1 above)
Comment (AndyEPhipps) [inline Gruntfile.js]: Good spot, thanks!

File: sass/base/components/partners/partners-grid.twig [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+                                <span>Find out more</span>
+                                </a>
+                            </div>
+                        </div>
+                    </article>
+                </li>
+            </ul>
+        </div>
+    </div>
+</div>
```
Comment (Heleen-cr) [inline sass/base/components/partners/partners-grid.twig]: looks like there's no new line there

**Pull request merged**